portfolio/src/main/java/com/google/sps/data/LoginStatus.java

```diff
@@ -0,0 +1,26 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.sps.data;
+
+/** An item on a comment list. */
+public final class LoginStatus {
+  public final Boolean isLoggedIn;
```

Comment:
nit: use boolean instead of Boolean. Always use the primitive rather than the object when possible

Reply:
main reasoning for using primitives:
Boolean variables can be null, true, or false
primitive boolean can only be true or false

it's better to avoid the nasty null pointer exception when possible. Boolean objects also take up more memory

Reply:
done.